### PR TITLE
Input

### PR DESCRIPTION
**Pull request by ArtemSuhoy**
Last changes inputs by Suhoy

### CODE REVIEW

File: scss/inputs/inputs-suhoy.scss [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+/* Airport axis */
+.input-xxxl{
+  font-family: inherit;
+  font-size: var(--fs-h4);
+  align-items: center;
+  border: var(--bw-3) var(--bs-solid) var(--color-transparent);
+  border-radius: var(--br-4);
+  line-height: var(--lh-h2);
+  padding: var(--p-sm) var(--p-md) var(--p-sm) var(--p-xxxxl);
+  width: 90%;
```
Comment (jazzkrop) [inline scss/inputs/inputs-suhoy.scss:27]: remove width

File: scss/inputs/inputs-suhoy.scss [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+/* Main page */
+.input-xxl{
+  font-family: inherit;
+  font-size: var(--fs-h4);
+  line-height: var(--lh-h4);
+  align-items: center;
+  border: var(--bw-3) var(--bs-solid) var(--color-transparent);
+  border-radius: var(--br-3);
+  padding: var(--p-input-xs) var(--p-md) var(--p-input-xs) var(--p-xxxxl);
+  width: 90%;
```
Comment (jazzkrop) [inline scss/inputs/inputs-suhoy.scss:58]: width

File: scss/inputs/inputs-suhoy.scss [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+.date-checker{
+  display: flex;
+  font-family: inherit;
+  font-weight: var(--fw-normal);
+  font-size: var(--fs-h4);
+  border: var(--bw-3) var(--bs-solid) var(--color-gray-default);
+  border-radius: var(--br-3);
+  padding: var(--p-xs);
+  line-height: var(--lh-h6);
+  width: 90%;
```
Comment (jazzkrop) [inline scss/inputs/inputs-suhoy.scss:121]: width

File: config.css [hunk 4]
```diff
@@ -45,7 +45,7 @@
 
   /* Color gray */
   --color-gray-default: rgba(217, 217, 217, 1); /* #d9d9d9 */
-  --color-gray-darken1: rgba(189, 189, 189, 1); /* #bdbdbd */
+  --color-gray---color-grdarken1: rgba(189, 189, 189, 1); /* #bdbdbd */
```
Comment (jazzkrop) [inline config.css]: strange name of color

**Review by jazzkrop — CHANGES_REQUESTED** [review]

**Review by olexandermushtyn — APPROVED** [review]
approve

**Pull request merged**